rskj-core/src/test/java/co/rsk/mine/TransactionModuleTest.java

```diff
@@ -404,8 +405,8 @@ private Web3Impl createEnvironment(Blockchain blockchain, MiningMainchainView ma
                 null,
                 configCapabilities,
                 null,
-                null
-        );
+                null,
+                Mockito.mock(Web3InformationRetriever.class));
```

Comment:
Passing `null` wouldn't work here?

Reply:
I guess It would, personally I don't like sending null dependencies